.github/workflows/test.yml

```diff
@@ -186,7 +205,22 @@ jobs:
       - name: Run tests
         timeout-minutes: 20
         run: |
-          cargo nextest run --package astria-bridge-withdrawer -- --include-ignored
+          cargo llvm-cov nextest --release \
+            --locked \
+            --package astria-bridge-withdrawer \
+            --run-ignored all \
+            --codecov --output-path codecov.json \
+            --no-clean
+      - name: Upload coverage to Codecov
+        uses: codecov/codecov-action@v5
+        with:
+          token: ${{ secrets.CODECOV_TOKEN }}
+          files: codecov.json
+          flags: doctest
+          fail_ci_if_error: true
+      - name: cleanup
```

Comment:
I added cleanup at the end to ensure that the potentially offending artifacts just don't get cached, and then skip cleanup when running.